src/test/java/core/basesyntax/service/impl/CsvReaderServiceImplTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.ReaderService;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class CsvReaderServiceImplTest {
+    private static final String SOURCE = "src/test/resources/source.csv";
+    private static final String EMPTY_SOURCE = "src/test/resources/empty.csv";
```

Comment:
consider using @ParametrizedTest and @CsvFileSource

Reply:
I can't use them. IDEA underlines it with red color and I can't import it. I don't know why, mb you could help me with it :)